Reset envorionment

In [126]:
rm(list=ls())

template

### **Spørgsmål  ()** 



## Opgave I

### **Spørgsmål I.1 (1)**

Hvad er sandsynligheden for ikke at finde nogle defekte varer?
Antager at fordeling følger en binorm fordeling.

In [127]:
pbinom(0,10,0.03)

[1] 0.7374241

Der er en 73% chance for ikke at finde fejl. 

Svar: **4**

### **Spørgsmål I.2 (2)**

Hvad er antallet af defekter, hvor der er 98% eller højere sandsynlighed for, at opnå dette antal
eller færre defekter i eksperimentet?

Vælger at teste alle svarmuligheder:

In [128]:
pbinom(0,10,0.03)
pbinom(1,10,0.03)
pbinom(2,10,0.03)
pbinom(5,10,0.03)
pbinom(8,10,0.03)

[1] 0.7374241

[1] 0.9654934

[1] 0.9972351

[1] 0.9999999

[1] 1

Der skal mindst to defekter til for at opnå 98% sandsynlighed.

Svar: **2**

### **Spørgsmål I.3 (3)** 

Hvilket af følgende udtryk beregner variansen?

## Opgave II

### **Spørgsmål II.1 (4)** 



### **Spørgsmål II.2 (5)** 



## Opgave III

### **Spørgsmål III.1 (6)** 

Hvis vi antager, at de sande standardafvigelser for de to metoder er ens, hvad er da estimatet
af den samlede (poolede) standardafvigelse?

Vil finde den poolede standardafvigelse ved først at regne den poolede varians. Herefter tager jeg kvadratroden for at finde std. afvigelsen. Se bogens s. 102

In [129]:
sigma = ((10 - 1) * 3.6^2 + (10 - 1) * 3.3^2) / (10 + 10 - 2)
sqrt(sigma)

[1] 3.453259

Jeg undersøger nu samtlige løsningsforslag:

In [130]:
(3.6+3.3)/2
sqrt((3.6^2+3.3^2)/2)
(3.6^2+3.3^2)/2
sqrt((3.6+3.3)/2)

[1] 3.45

[1] 3.453259

[1] 11.925

[1] 1.857418

Svar: **2**

### **Spørgsmål III.2 (7)** 

Hvilket udsagn er korrekt (begge dele af udsagnet skal være korrekt)?

Nogle kloge ord her.

### **Spørgsmål III.2 (7)** 

Hvad er det mindste antal observationer, man skal tage fra hver gruppe for at opfylde ovenstående
krav?

Man kan enten estimere det i hånden eller lade R om det. Vi forsøger med R:

In [131]:
power.t.test(power=0.98, delta=5, sd=sqrt(16), sig.level=0.01, type="two.sample")


     Two-sample t test power calculation 

              n = 29.15117
          delta = 5
             sd = 4
      sig.level = 0.01
          power = 0.98
    alternative = two.sided

NOTE: n is number in *each* group


Læg mærke til at vi skal bruge standard-afvigelsen, derfor tager vi sqrt() af variansen, 16. Husk at vi her har med en two sample test at gøre.
Fra outputtet skal vi bruge mindst 30 observationer.

Svar: **3**

## Opgavae IV

### **Spørgsmål IV.1 (8)** 

Hvis komponenternes gennemsnitlige levetid er 3 år, hvilken af følgende R-koder beregner sand-
synligheden for, at en tilfældigt udvalgt komponent holder længere end et år?

forklarende tekst

In [132]:
code

ERROR: Error in eval(expr, envir, enclos): object 'code' not found


### **Spørgsmål IV.2 (10)** 

Hvad er antallet af observationer i stikprøven?

idk hvordan vi skal afgøre det ud fra opgaven alene.

### **Spørgsmål IV.3 (11)** 

Baseret på denne analyse, hvad er konklusionen på en test af nulhypotesen

Blank

### **Spørgsmål IV.3 (11)** 

Baseret på denne analyse, hvad er konklusionen på en test af nulhypotesen

## Opgave V

### **Spørgsmål V.1 (12)** 

eregnet med simulering, hvad er standardafvigelsen ... hvor X1 og X2 er uafhængige og begge følger en standard normalfordeling?

Vi bruger lidt simulation.

In [ ]:
x <- 1000000
x1 <- rnorm(x,0,1)
x2 <- rnorm(x,0,1)
Y  <- exp(x1) + x2^4 + x1*x2
sd(Y)

[1] 10.01407

Svar: **2**

### **Spørgsmål V.2 (13)**

pas

## Opgave VI

### **Spørgsmål VI.1 (14)**

Hvad er estimatet af \beta_1?

In [ ]:
# Mangankoncentrationer
x <- c(0, 0, 2, 2, 4, 4, 6, 6, 8, 8, 10, 10)
# ICP-AES værdier
y <- c(114, 14, 870, 1141, 2087, 2212, 3353, 2633, 3970, 4299, 4950, 5207)
# Estimer multipel lineær regressionsmodel
fit <- lm(y ~ x)

# Vis estimerede parametre mm.
summary(fit)


Call:
lm(formula = y ~ x)

Residuals:
    Min      1Q  Median      3Q     Max 
-442.16 -120.98   42.65  122.27  277.84 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept)    49.19     106.69   0.461    0.655    
x             504.33      17.62  28.624  6.3e-11 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 208.5 on 10 degrees of freedom
Multiple R-squared:  0.9879,	Adjusted R-squared:  0.9867 
F-statistic: 819.3 on 1 and 10 DF,  p-value: 6.304e-11


Fra outputtet ser vi at Estimate for x (beta_1) er 504.33.

Svar: **2**

### **Spørgsmål VI.2 (15)**

Hvad er 95% prædiktionsintervallet ved denne koncentration?

Indsætter en prædiktion i modellen.

In [ ]:
predict(fit, newdata=data.frame(x=5), interval="prediction", level=0.95)

fit      lwr      upr     
1 2570.833 2087.363 3054.303

**NOTE!** Husk at angive newdata som en data.frame

Svar: **1**

### **Spørgsmål VI.3 (16)**

Hvad er 95% prædiktionsintervallet ved denne koncentration?

### **Spørgsmål VI.4 (17)**

Hvilket af følgende udsagn er den korrekte fortolkning af disse to plots?

qq plottet er flot lineært. Residualplottet er dog ikke tilfældigt fordelt, som det ellers burde det.

Svar: **2**

### **Spørgsmål VI.5 (18)**

Hvilket af de følgende udsagn er korrekt, givet et signifikansniveau på α = 1%?

Mulighed 2 og 3 er forkerte, idet en 1ppb ville forøge med:

In [ ]:
12.6469 + 553.4783 * 1 - 5.5157 * 1^2

[1] 560.6095

Mulighed 4 giver ikke mening.

Svar: **5**

## Opgave VII

### **Spørgsmål VII.1 (19)**

Hvilket af de følgende udsagn er korrekt, givet et signifikansniveau på α = 1%?

### **Spørgsmål VII.2 (20)**

Hvad er den korrekte konklusion om nulhypotesen når der testes på signifikansniveau α = 0.05?

Nulhypotesen accepteres idet p værdien er over signifikantsniveauet.

Svar: **3**

### **Spørgsmål VII.3 (21)**

Hvad er det kritiske niveau, dvs. en χ2 -fraktil, for test af nulhypotesen på signifikansniveau α = 0.01?

In [ ]:
qchisq(p=1 - 0.01, df=(3-1)*(2-1))

[1] 9.21034

Svar: **1**

## Opgave VIII

### **Spørgsmål VIII.1 (22)
Hvilket af følgende udsagn er korrekt, når man anvender signifikansniveau α = 0.05 (både
argument og konklusion skal være korrekt)?

In [ ]:
#   mu ± var/sqrt(n)  * qt(1 - alpha, df = n - 1)
c(21.5 - 9.8/sqrt(56) * qt(0.975,df=55),
  21.5 + 9.8/sqrt(56) * qt(0.975,df=55))

[1] 18.87554 24.12446

23 ligger i 95% intervallet derfor kan udsagnes underbygges.

Svar: **1**

### **Spørgsmål VIII.2 (23)
Hvad er værdien af teststatistikken som bruges til at teste hypotesen?

In [ ]:
(21.5 - 23) / (9.8 / sqrt(56))

[1] -1.145405

tobs er udregnet til -1.145. 

Svar: **3**

## Opgave IX

### **Spørgsmål IX.1 (24)**
Hvad er sandsynligheden for, at der er to eller flere besøgende på hjemmesiden i en tilfældigt udvalgt periode på et minut?

Bruger Poisson distribution (number of events per interval). 
Bruger dermed ppois og undersøger sandsynligheden for at finde flere end 1 -> altså 2 eller højere

lambda er lig 7, idet der er 7 besøgende per minut.

In [ ]:
1 - ppois(1, lambda=7)

[1] 0.9927049

Svar: **5**

### **Spørgsmål IX.2 (25)**
Hvad er sandsynligheden for, at der ikke er besøgende i en tilfældigt udvalgt periode på 30 sekunder?

lambda er nu 7/2, idet vores interval er 30 sekunder. Undersøger det tilfælde hvor der kun er 0.

In [ ]:
ppois(0, lambda=7/2)

[1] 0.03019738

Svar: **1**

## Opgave X

### **Spørgsmål X.1 (26)
Hvad er total sum of squares (SST), treatment
sum of squares (SS(Tr)) og sum of squared errors (SSE)?

### **Spørgsmål X.2 (27)**


## Opgave XI

### **Spørgsmål XI.1 (28)**


### **Spørgsmål XI.2 (29)**


### **Spørgsmål XI.3 (30)**
